exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,8 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+let totalNumber = numberOfMentors + numberOfStudents;
+let percentageOfStudents = (numberOfStudents / totalNumber) * 100;
+let roughPercentageOfStudents = Math.round(percentageOfStudents);
```

Comment:
Why did you decide to call this variable `roughPercentageOfStudents` in this case - what makes it rough?